### ISSUE TITLE
Test folder not loaded when I accept to modify the script after creating the test

### ISSUE THREAD

**Issue by paulinedessingue-lm** [cerberustesting/cerberus-source#2220]
Hi Cerberus team !

I have tested the new version v4.10 and I notice a problem : 
1. I create a testcase
2. I fill the testcase header and I save
3. A window proposes me to edit the test script right away, I accept
4. In the script page, the testfolder is not loaded
--> I can't add a step, add a description, add an action or a property.

Can you help me ?

![Cerberus4-10-testfoldernotloaded](https://user-images.githubusercontent.com/54071709/100471144-f4ead800-30d9-11eb-88f2-795f00b42414.jpg)

**Comment by vertigo17** [comment]
I got a similar feedback from Luigi SECO. ;-(.
Can you CTRL+F5 on both the Testcase List page and Testcase Script page ?
I think this is a cache issue as I don't manage to reproduce.

**Comment by paulinedessingue-lm** [comment]
Hi Benoit, thanks for your answer !

The testcase is well created and it's visible in test folder. 
I tried to Ctrl+F5 or do it in private navigation, but the test folder still doesn't appear in Script page.
I checked the logs, but found nothing of significance.

Here, it's my screen recording, if it can help.. maybe I do something wrong ?
[Test Cases - Google Chrome 2020-11-30 11-11-54.zip](https://github.com/cerberustesting/cerberus-source/files/5615344/Test.Cases.-.Google.Chrome.2020-11-30.11-11-54.zip)
